gpt-oss-on-intel-xeon.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Running GPT OSS on Intel Xeon CPU"
+thumbnail: /blog/assets/xxx.png # waiting for upload
+authors:
+- user: Jiqing
+  guest: true
+  org: Intel
+- user: MatrixYao
+  guest: true
+  org: Intel
+- user: kding1
+  guest: true
+  org: Intel
+---
+
+
+# Running GPT-OSS on Intel Xeon CPUs
```

Comment:
Let's emphasize GCP C4, So, the name may better be `Optimizing GPT OSS on 6th Gen Xeon at GCP` ? @kding1